### ISSUE TITLE
The DestinationTableName cannot be null or empty. You must specify a valid DestinationTableName.

### ISSUE THREAD

**Issue by jrt324** [zzzprojects/EntityFramework-Extensions#18]
```
              var db = new TestContext();
                List<LoginLogEntity> users = new List<LoginLogEntity>(){new LoginLogEntity
                    {
                        Id = Guid.NewGuid(),
                        LoginType = 1,
                        LoginDevice = "IOS",
                        LoginIP = "",
                        LoginStatus = 1,
                        LoginPlace = "",
                        LoginTime = DateTime.Now,
                        OrgId = "971",
                        RealName = "JRT1",
                        UserId = userId,
                        UserType = 1
                    },
                     new LoginLogEntity
                    {
                        Id = Guid.NewGuid(),
                        LoginType = 1,
                        LoginDevice = "IOS",
                        LoginIP = "",
                        LoginStatus = 1,
                        LoginPlace = "",
                        LoginTime = DateTime.Now,
                        OrgId = "971",
                        RealName = "JRT1",
                        UserId = userId,
                        UserType = 1
                    }
               };
              db.BulkInsert(users);
```

Test code work's fine in my local server,but put it on Remote server, it always show me follow exception:

The DestinationTableName cannot be null or empty. You must specify a valid DestinationTableName.
   at ..(BulkOperation )
   at ..(BulkOperation )
   at Z.EntityFramework.Extensions.EntityBulkOperation`1.BulkInsert()
   at ..[](BulkOperation`1 , DbContext , List`1 , Boolean , List`1 , Type , String )
   at ...(SchemaEntityType )
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at ..[](DbContext , BulkOperation`1 , IEnumerable`1 , List`1)
   at ..[](BulkOperation`1 , DbContext , IEnumerable`1 , List`1)
   at DbContextExtensions.BulkInsert[T](DbContext this, IEnumerable`1 entities, Action`1 bulkOperationFactory)
   at DbContextExtensions.BulkInsert[T](DbContext this, IEnumerable`1 entities)

**Comment by zzzprojects** [comment]
I believe this issue is caused because the remote server uses an unsupported provider.

Can you report the connection provider which is currently in use?

```csharp
context.Database.Connection.GetType().FullName
```

We will add a new custom error when a similar issue happens to make easier to understand what's happening.

**Comment by jrt324** [comment]
I used [Entity Framework Profiler ](https://www.hibernatingrhinos.com/products/efprof) and Glimpse in my project.

Follow is the `context.Database.Connection.GetType().FullName ` result:

HibernatingRhinos.Profiler.Appender.ProfiledDataAccess.ProfiledConnection`1[[Glimpse.Ado.AlternateType.GlimpseDbProviderFactory`1[[System.Data.SqlClient.SqlClientFactory, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]], Glimpse.Ado, Version=1.7.3.0, Culture=neutral, PublicKeyToken=null]]

**Comment by zzzprojects** [comment]
Thank you, both providers are supported:
- HibernatingRhinos
- Glimpse

But they cannot be used at the same time currently.

The code is looking if the InnerConnection if a SqlConnection but instead a Glimpse connection is found!

This issue will be fixed at the beginning of the week.

**Comment by jrt324** [comment]
Is this issue fixed in version 3.10.13?

**Comment by zzzprojects** [comment]
Nope, we will try to release the fix this night or tomorrow morning.

**Comment by zzzprojects** [comment]
Hello @jrt324 ,

The latest version v3.10.15 should fix this issue.

Unfortunately, we didn't have the time to test it since we had some problem to reproduce this scenario so we will need your help here to make sure this is now working.

Let us know if the new version has fixed this issue.

Best Regards,

Jonathan

**Comment by zzzprojects** [comment]
Closing Issue: Fixed in v3.10.15
